packages/vue/src/T.ts

```diff
@@ -20,14 +20,21 @@ export const T = defineComponent({
     ns: { type: String as PropType<NsType> },
     language: { type: String as PropType<string> },
   },
-  setup() {
+  setup(props, context: SetupContext) {
+    const { slots } = context;
+    const { params } = props;
+    const slotsParams = {};
+    Object.keys(slots).forEach((key) => {
+      slotsParams[key] = slots[key]();
+    });
+    const assignedParams = Object.assign({}, params, slotsParams);
```

Comment:
Cool, this looks a lot cleaner 👍 I just wonder, if this shouldn't be in `computed` method, e.g.:

```
const assignedParams = computed(() => {
  const slotsParams = {};
  Object.keys(slots).forEach((key) => {
    slotsParams[key] = slots[key]();
  });
  return Object.assign({}, params, slotsParams);
});
```

So that if slots are changed, the params are also changed.